src/ILLink.RoslynAnalyzer/TrimAnalysis/ArrayValue.cs

```diff
@@ -9,7 +9,12 @@ namespace ILLink.Shared.TrimAnalysis
 {
 	partial record ArrayValue
 	{
-		public ArrayValue (SingleValue size) => Size = size;
+		public SingleValue ElementType;
+		public ArrayValue (SingleValue elementType)
```

Comment:
I think we could make `ElementType` an `ITypeSymbol` in the roslyn-specific code, similar to how it's a `TypeReference` in the linker code.

Reply:
As noted above, for this change we don't need to handle array values yet... so I would not try to solve it right now.